core/src/main/java/io/questdb/cairo/VarcharTypeDriver.java

```diff
@@ -33,30 +33,54 @@
 import static io.questdb.cairo.ColumnType.VARCHAR_AUX_SHL;
 
 public class VarcharTypeDriver implements ColumnTypeDriver {
+    public static final int HEADER_FLAG_NULL = 4;
```

Comment:
The naming in this class uses the `VARCHAR_` prefix for public constants, so they're easy to use with static imports.

Reply:
Renamed in 31bb6c3f990ec9242b3b531c359b1dd942ef02af